amnesia.py

```diff
@@ -9,6 +9,12 @@
 from tweepy.error import TweepError
 
 
+# By default, do not actually delete any tweet/like.
+# Just print out what will be deleted.
+# Set this to False to actually delete tweets/likes.
+DRY_RUN = True
```

Comment:
We have a `dry_run` value in `configs-sample.py`. This way it can be overridden in `configs.py`.

Since we've already set dry run, implicitly, to false I think we should keep it that way and allow it to be set to True in the config.

```
dry_run = True
```